### ISSUE TITLE
Staging Review finding: Error state heading has the wrong heading level

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82715]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management

**Product:** Appoint a Representative MVP + Representative Status Widget MVP
**Feature:** Appoint a Representative MVP + Representative Status Widget MVP

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content that doesn't match other content on VA.gov.](https://design.va.gov/about/experience-standards/#user-encounters-content-that-is-similar-to-other-content-on-vagov)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the error state (test user +2), the error message has an H2 heading. This doesn't match the heading structure of the page when there's not an error state.

## Recommended action

Since it occupies the same space as the current accredited representative card and is notifying users that the card can't be displayed, the error alert should have an H3 instead of an H2.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
The H2 wasn't intentional, to fix, we simply need to update the error state so that it is using the dynamic header as expected.
